src/Analyzers/CSharp/CodeFixes/AddInheritdoc/AddInheritdocCodeFixProvider.cs

```diff
@@ -78,21 +66,15 @@ public override async Task RegisterCodeFixesAsync(CodeFixContext context)
 
     protected override async Task FixAllAsync(Document document, ImmutableArray<Diagnostic> diagnostics, SyntaxEditor editor, CancellationToken cancellationToken)
     {
-        string? newLine = null;
-        SourceText? sourceText = null;
+        var options = await document.GetLineFormattingOptionsAsync(cancellationToken).ConfigureAwait(false);
+        var newLine = options.NewLine;
+        var sourceText = await document.GetValueTextAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
we only got here because we want to fix things.  deferring any of this stuff is pointless.  the compiler already said there was a problem.